### ISSUE TITLE
Adding Parts Attribute

### ISSUE THREAD

**Issue by mbprondzinski** [ArctosDB/arctos#2294]
When adding a Parts Attribute, is it necessary to include a part name?  "Location" of specimen was set up under Parts Attribute "add more."  When I click on "add specimen part" I get this screen:
![image](https://user-images.githubusercontent.com/43214224/66236982-78b55a00-e6b9-11e9-914c-445b2a7b6a5a.png)
"Attribute Type" has "location" and the" Value" is the location where the specimen is stored.  The actual part name was recorded in the main "parts" grid.  Does it have to be entered in the "add more" grid as well?   Thanks.

**Comment by Jegelewicz** [comment]
@mbprondzinski where in Arctos are you at this screenshot?

**Comment by mbprondzinski** [comment]
…data entry (the green screen)  Bing wanted to know!

**Comment by Jegelewicz** [comment]
Well, he's got me beat. I have no idea how he got that screen and it appears he has discovered something I knew nothing about...

@dustymc can you enter part attributes along with the part in data entry? If so, how is this done?

**Comment by dustymc** [comment]
Yep, that's from the 'add more' button on data entry.

Attributes are children of parts and they must be entered together - you can use that screen instead of the normal/embedded screen to create specimens with parts with attributes.

**Comment by Jegelewicz** [comment]
I had no idea...and I still don't see how you get to this screen. When I select "add more parts" I don't get anything like this.

Also, Mary Beth set this record to load yesterday, so why is it still hanging around in the "browse and edit" area?

**Comment by Jegelewicz** [comment]
OK, NOW I see - but I have to scroll down to find that thingee - I have no idea how Bing found it!

After selecting "Add more parts, this is EXACTLY how my screeen looks:
![image](https://user-images.githubusercontent.com/5725767/66240111-e4e38e00-e6b8-11e9-9a98-5a3f0caa1548.png)

I have to scroll to find the tool:
![image](https://user-images.githubusercontent.com/5725767/66240142-fe84d580-e6b8-11e9-9980-8cdf246d32f4.png)

And scroll some more!
![image](https://user-images.githubusercontent.com/5725767/66240170-1c523a80-e6b9-11e9-9ca0-5d40213a9d21.png)

OH! There it is!

Is there any way we can avoid all of this wasted grey space? If I hadn't seen Mary Beth's screen shot and been persistent in looking, I never would have known anything happened when I selected "Add more parts".

**Comment by Jegelewicz** [comment]
So, from now on when Bing is entering parts, he should ignore the "parts" section and use the "Add more parts" tool so that he can add the part locations along with the part. @mbprondzinski let me know if that is clear as mud.

**Comment by dustymc** [comment]
Oh my...

I can't make it that ugly on my phone!

"Full-stack developer with a focus on CSS layout" might not be a bad place to start that SABI proposal.

FWIW here's what it looks like on my laptop.

<img width="1381" alt="Screen Shot 2019-10-04 at 2 24 52 PM" src="https://user-images.githubusercontent.com/5720791/66240969-c7132a80-e6b2-11e9-91bb-3b4e934b8557.png">

**Comment by Jegelewicz** [comment]
Well WTH? Why so crappy on mine? Kinda a bummer because that is really useful. However, it isn't documented anywhere, so there is that...

**Comment by campmlc** [comment]
It doesn't look that good on my laptop! I've always wondered why the screen
is so squished.

**Comment by mbprondzinski** [comment]
Ha-ha-ha!  Don't worry Teresa, you are still the Arctos Queen and Bing is still an Arctos Luddite. He was still obsessing over the "location" this morning and being difficult, so I said I would figure it out..and voila!  But I did remember that we can't have Arctos assign a catalog number until we get the Invert Paleo into Arctos?  Or can we tell Arctos what number to start numbering at???  Also, I agree that the location of the "add more" tab could be more accessible!
________________________________

**Comment by dustymc** [comment]
Also, to be clear, if launching specimens towards the shelves with a potato gun is a "1," and fully embracing barcodes is a 10, then 'part location' is maybe a 4 (and tracking this through taxonomy about "2"). If it's possible to do whatever you're doing through barcodes, future-you will be thankful. If it isn't, part location is the best alternative. (Which still doesn't make it very good!)

**Comment by Jegelewicz** [comment]
No potato launchers here - and it would be super easy to use barcodes in this case, but there just isn't the person to set it all up and apply them to the shelves/boxes. This attribute is our best alternative until such time there is a person/budget to use barcodes.

**Comment by mbprondzinski** [comment]
I take umbrage on this issue!  We couldn't possibly be THE ONLY institution in Arctos that isn't barcoding their cabinets/shelves?!  Yes, we could, but we don't have the time or capacity to do that at this time.  And no, we don't use "potato guns" to shelve items.

**Comment by dustymc** [comment]
>THE ONLY

No, not even close. Reality has a way of doing that. Part location is (I think) a pretty good solution when you find yourself there, but it's also not something that I'd ever see as ideal.

**Comment by mbprondzinski** [comment]
I can live with it...so can Bing (!)

**Comment by Jegelewicz** [comment]
@dustymc it is unclear to me how the "Add Parts" thingee functions. Is this really just adding part attributes (so the part needs to be included in the regular form) OR does this function add both the part and any attributes you assign to it?

**Comment by dustymc** [comment]
All of the add tools just normalize using various bulkloaders - that one writes to http://arctos.database.museum/tools/BulkloadParts.cfm, which needs the part and accompanying attributes as a row.

**Comment by Jegelewicz** [comment]
Needs a How To
